src/config.rs

```diff
@@ -65,31 +65,16 @@ pub struct Config<C: GKRConfig> {
     pub gkr_scheme: GKRScheme,
 }
 
-impl Config<M31ExtConfig> {
+impl<C: GKRConfig> Config<C> {
     pub fn new(gkr_scheme: GKRScheme) -> Self {
         Config {
-            field_size: 93,
+            field_size: C::ChallengeField::FIELD_SIZE,
```

Comment:
I am being lazy here and derive the field size from challenge field, which the field eventually should fall into - but feel free to debate about the slacking bits from a slightly larger field bits, I am open to changes here.